dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogConfigBuilderTest.kt

```diff
@@ -179,32 +135,23 @@ class DatadogConfigBuilderTest {
                     fakeClientToken,
                     fakeApplicationId,
                     DatadogEndpoint.RUM_US,
-                    DatadogConfig.DEFAULT_SERVICE_NAME,
                     fakeEnvName
                 )
             )
     }
 
     @Test
-    fun `builder with custom service name`(
```

Comment:
Why was this test was removed ?

Reply:
Oups.....I lost this during a rebase, I thought I put it back. Good spot.